src/registrar/templates/emails/transition_domain_invitation.txt

```diff
@@ -1,20 +1,42 @@
 {% autoescape off %}{# In a text file, we don't want to have HTML entities escaped #}
-Hi.
+Hello from .gov.
 
-You have been added as a manager on {% if domains|length > 1 %}multiple domains (listed below){% else %}{{ domains.0 }}{% endif %}.
+Our new domain management system—the .gov registrar—is now available. The organization, contacts, and DNS information for your .gov domain{% if domains|length > 1 %}s{% endif %} have been transferred to our new registrar. 
+
+Your .gov domain{% if domains|length > 1 %}s{% endif %}:
+{% for domain in domains %} - {{ domain }}
+{% endfor %}
+If you’re not affiliated with the above domain{% if domains|length > 1 %}s{% endif %} or think you received this message in error, let us know in a reply to this email. 
+
+
+CREATE A LOGIN.GOV ACCOUNT
+
+You can’t use your old credentials to access the new registrar. Access is now managed through Login.gov, a simple and secure process for signing into many government services with one account. Follow these steps to create your Login.gov account <https://login.gov/help/get-started/create-your-account/>. 
+
+When creating an account, you’ll need to provide the same email address you used to log in to the old registrar. That will ensure your domains are linked to your Login.gov account. 
+
+If you need help finding the email address you used in the past, let us know in a reply to this email.
+
+
+CHECK YOUR .GOV DOMAIN CONTACTS
+
+This is a good time to check who has access to your .gov domain{% if domains|length > 1 %}s{% endif %}. The admin, technical, and billing contacts listed for your domain{% if domains|length > 1 %}s{% endif %} in our old system also received this email. In our new registrar, these contacts are all considered “domain managers"
```

Comment:
needs a period at end: "domain managers**.**"

Reply:
Nice catch!

Reply:
@abroddrick sorry, also noticing the use of a prime/hatch mark at the end of domain managers when it should be a quotation mark. ”

Reply:
once again this bites me. any chance you know a hot key to make sure it's a quotation mark?

Reply:
option/alt + shift + '